Makefile

```diff
@@ -105,6 +105,7 @@ quick-reset:
 	bash ops/db.sh 'truncate table onchain_transaction cascade;'
 	bash ops/db.sh 'truncate table rebalance_profile cascade;'
 	bash ops/db.sh 'truncate table app_instance cascade;'
+	bash ops/redis.sh 'flushall'
```

Comment:
Redis is used a lot more heavily now, as such it may contain stale values that need to be removed during resets.